pkgs/applications/misc/glom/default.nix

```diff
@@ -93,7 +91,8 @@ in stdenv.mkDerivation rec {
     evince
     isocodes
     python3.pkgs.pygobject3
-    gtksourceviewmm4
```

Comment:
Hmm, weird that they did not include the gtksourceview bump in the release.

But it indeed looks like it is newer than fa5ff04f, compare commits in [current master](https://gitlab.gnome.org/GNOME/glom/-/commits/08299548799c31b219833b335f816f93620bd16a) with the [release](https://gitlab.gnome.org/GNOME/glom/-/commits/1.32.0).